src/screens/SignupScreen.tsx

```diff
@@ -203,13 +205,14 @@ export const SignupScreen: React.FC<SignupProps> = () => {
 
 const styles = StyleSheet.create({
     scrollviewContainer: {
-        paddingTop: StatusBar.currentHeight
+        paddingTop: StatusBar.currentHeight,
+        flex: 1
     },
     contentContainer: {
         alignItems: 'center',
         paddingHorizontal: 20,
-        height: '100%',
-        paddingTop: '8%',
+        flex: 1,
+        paddingTop: '20%',
```

Comment:
what was the difference flex here instead of height 100 , I'm just curious

Reply:
Flex 1 uses 100% of the view. Hehehe, just realized that it is the same thing